src/analytics/Events.tsx

```diff
@@ -448,6 +448,10 @@ export enum FiatExchangeEvents {
   cico_fc_link_account_back = 'cico_fc_link_account_back',
   cico_fc_link_account_provider_website = 'cico_fc_link_account_provider_website',
 
+  // Fiat Connect link KYC & account page
+  cico_fc_link_kyc_account_back = 'cico_fc_link_kyc_account_back',
+  // persona_kyc_start is fired when the 'Set up ID Verification' button is clicked
```

Comment:
is this comment intentional here?

Reply:
yes it is, just in case somebody was looking right here for what happens when the user clicks on the CTA on this page.